lib/store/reducer.js

```diff
@@ -17,10 +17,13 @@ const initialState = {
     state: null,
     value: null,
   },
+  location: {
+    pos: null,
+    isWatching: false,
+  },
```

Comment:
A pattern I'm introducing with my pending redux patch is async values having `status` and `value` properties. The `value` property varies depending on the status.

```js
{
  location: {
    status: 'STARTED' | 'STOPPED' | 'ERROR'?
    value: null | { latitude: ..., longitude: ... }
  }
}
```

Although I guess this is slightly different as the `status` doesn't directly map to a corresponding value. For example `'STARTED'` doesn't mean that we have a value yet.